hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This release adds the strategy :func:`~hypothesis.extra.numpy.integer_array_indices`,
+which generates tuples of integer arrays that can be used to perform `advanced indexing <http://www.pythonlikeyoumeanit.com/Module3_IntroducingNumpy/AdvancedIndexing.html#Integer-Array-Indexing>`_
```

Comment:
I think this would be unclear to people who don't use the data stack - how about "tuples of Numpy arrays" (dtype not really important here) and "into a Numpy array of a specified shape"?